src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -576,6 +576,21 @@ public function compileRandom($seed)
     }
 
     /**
+     * Compile the field database function into SQL.
+     * @param string $column
+     * @param array $columnValues
+     * @return string
+     */
+    public function compileField($column, $columnValues)
+    {
+        $columnValues = collect($columnValues)->map(function ($columnValue) {
```

Comment:
Avoid using helpers inside the core, use new Collection instead and seems like ->transform() fits better here

Reply:
I was basically following the same rules that were used throughout the Grammar class. If you check the source you'll notice that the "collect()" helper function is used quite a lot.

Anyway, I agree that transform may be a better fit in this instance.

Reply:
I believe that at some point we agreed to not use helpers in the core, but I could be mistaken.